examples/functions/stopPlace/Netex_10_StopPlace_withParking_1.xml

```diff
@@ -1017,6 +1017,7 @@ Changes
 										<EntranceEquipment version="any" id="nptg:9100WIMBLDN_A2b-EI5_e1">
 											<Door>true</Door>
 											<KeptOpen>true</KeptOpen>
+											<DoorControlElementHeight>0.8</DoorControlElementHeight>
```

Comment:
Just asking to the group: do we want to change this to SI_METER now? While the other element is now in centimeter?

Reply:
will move everything to m => https://github.com/NeTEx-CEN/NeTEx/issues/610